service/docs/source/geopm_agent_cpu_activity.7.rst

```diff
@@ -115,11 +115,11 @@ be used to force the entire application to run at a fixed frequency.
       Used to build a mapping of uncore frequencies to maximum
       memory bandwidths for frequency steering.
 
-Policy Generation
------------------
+ConstConfig Generation
```

Comment:
I'm debating on how to call the configuration file, besides how it's being proposed now (ConstConfig file):
- Simply: configuration file
- ConstConfigIOGroup configuration file
- Characterization file
- Constant configuration file
- ...